fix: Adjust link for starter project repo

princessruthie opened the pull request:
(Excited for the class!!)

princessruthie committed:
fix: Adjust link for starter project repo

atroutt commented on index.html:
```diff
@@ -228,8 +228,9 @@
-            <pre><code class ="bash">git clone git@github.com:atroutt/android-beginners.git</code></pre>
+            <p class="left-align" style="margin-top:40px">Alternatively, 
+		    the repo using git:</p>
+            <pre><code class ="bash">https://github.com/atroutt/AndroidBeginnerFirstApp.git</code></pre>
```
This, github.com/atroutt/AndroidBeginnerFirstApp, is the finished app, but I put a copy of the starter app here in the android-beginners repo. We will use the starter app in the class and work toward implementing the finished app.

I see you are using the https url instead of ssh, that's probably simpler. In that case, the url should be `https://github.com/atroutt/android-beginners.git`

atroutt commented:
Thank you for going through this!! ❤️

princessruthie committed:
fix: Link to starter project not finished one.

(And today is the day I learned I never set up ssh on this machine.  Thanks for that!)
